### ISSUE TITLE
Network not accessible on single host due to too many connections

### ISSUE THREAD

**Issue by lingzhang-lyon** [rancher/rancher#10411]
Some times the network on a single host not working properly,  containers on this host could not be reached for a short period randomly. 

we found the network-manger on this host got following error
 ```
11/27/2017 9:05:28 AMtime="2017-11-27T17:05:28Z" level=info msg="CNI down" cid=b501a06ad7639c5af6e0c7c2d803b32ecaed5ad22e7589237e953f4824154408 networkMode=ipsec
11/27/2017 9:05:29 AMtime="2017-11-27T17:05:29Z" level=error msg="vethsync/utils: error deleting dangling veth: &{{8707 1500 0 vethrb501a06ad7 4e:73:da:cf:93:5b up|broadcast|multicast 69699 3 3 <nil>  0xc42502a0e8 0 <nil> ether} }"
11/27/2017 9:05:43 AMtime="2017-11-27T17:05:43Z" level=info msg="Setting up resolv.conf for ContainerId [7045e552225bf828374693da2aa1c240fe73840dd0d62c6a32b336408f225a1a]"
11/27/2017 9:05:43 AMtime="2017-11-27T17:05:43Z" level=info msg="CNI up" cid=7045e552225bf828374693da2aa1c240fe73840dd0d62c6a32b336408f225a1a networkMode=ipsec
11/27/2017 9:05:45 AMtime="2017-11-27T17:05:45Z" level=info msg="CNI up done" cid=7045e552225bf828374693da2aa1c240fe73840dd0d62c6a32b336408f225a1a networkMode=ipsec result=IP4:{IP:{IP:10.42.26.120 Mask:ffff0000} Gateway:10.42.0.1 Routes:[{Dst:{IP:169.254.169.250 Mask:ffffffff} GW:<nil>} {Dst:{IP:0.0.0.0 Mask:00000000} GW:10.42.0.1}]}, DNS:{Nameservers:[] Domain: Search:[] Options:[]}
11/27/2017 9:19:05 AMtime="2017-11-27T17:19:05Z" level=info msg="CNI down" cid=7045e552225bf828374693da2aa1c240fe73840dd0d62c6a32b336408f225a1a networkMode=ipsec
```

This issue happened several times on different hosts within our cluster. 
Each time there are some services which have many connections from other hosts were scheduled on the host  that has unstable network issue. 

We assume maybe it reach some kind of connection limit on particular host and the health-check agent on that host is monitoring too many backends  causing the network unstable . so we disable the health-check agent on that host, it kind fixed the problem on that host.

Just wondering do we actually have limit of connections on each host that may causing network-mangager unstable? how we can check/increase the limit?

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v1.6.10` Cattle: `v0.183.22` UI: `v1.6.23` |
|Access|`github` `admin`|
|Orchestration|`Cattle`|
|Route|`service.containers`|

**Comment by aemneina** [comment]
Are your services seeing a high load (how many sustained connections)? Your service container might need tuning to accept more connections. That service might be overloaded and might not be able to respond to the health checks.

**Comment by lingzhang-lyon** [comment]
hi aemneina,

Thank you for your reply. 

Yeah, on that host we have a service (A) that have ~500 connections. 
But looks like  other services (B, C, D ....) on the same host, which don't have many connections on each container, also failed health-check randomly. (Totally number of these services is about 70.)

We assumed high totally connection number on the host make the `network-manager` not function properly, since `healthcheck` agent also have a lot of connections, so we disabled it on that host and the situation was alleviated a lot.  Reducing health check interval could also help .

Later on we also tried to scheduled service A to a separate host, and the network on the original host seems stable now.

But still wondering why `network-manager` got unstable (or something else that make the whole network on the host not work,  affecting all containers not only one ) when total connection on host is high?

another question, how can we check the actual backends number the `healthcheck` agent is monitoring?

Thanks again for the help.
